lib/http_fuzzer/engine.py

```diff
@@ -278,14 +274,17 @@ def request_without_data(request, req_type, retries, time_sleep, timeout_sec, pa
 
     """
     request_line, headers_alone = request.split('\r\n', 1)
-    headers = Message(StringIO(headers_alone)).dict
+    headers = message_from_string(headers_alone)
     clean_headers = {x.strip(): y for x, y in headers.items()}
     headers = clean_headers
     headers.pop("Content-Length", None)
     url_sample = request_line.strip().split(' ')[1]
     for target in targets:
         url = url_sample.replace('__target_locat_here__', str(target))
-        port = url[url.find(':', 7) + 1:url.find('/', 7)]
```

Comment:
Also there was mitake in finding port used from the url. I fixed it. @securestep9 Please check it and let me know for any further corrections if any :)